vocode/streaming/synthesizer/azure_synthesizer.py

```diff
@@ -201,8 +203,11 @@ def create_ssml(
         return ElementTree.tostring(ssml_root, encoding="unicode")
 
     def synthesize_ssml(self, ssml: str) -> speechsdk.AudioDataStream:
```

Comment:
I wanted to just use functools' @lru_cache decorator - but `self` is a new instance every invocation, and the chain of `self` for each parent is also a new instance all the way up. so I created the `AzureSSMLCacheManager` below instead to achieve the same effect